src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -7,24 +7,70 @@
 public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
-
+    public String command= new String();
```

Comment:
Wow, your enhancement look really neat and easy to read.
One question, what do you think if the command attribute is set to private?

Reply:
Missing empty space before `=`. If possible, use `private` instead of `public`.